diff --git a/node_modules/@react-navigation/native/lib/commonjs/index.js b/node_modules/@react-navigation/native/lib/commonjs/index.js
index fa744ed..32daefc 100644
--- a/node_modules/@react-navigation/native/lib/commonjs/index.js
+++ b/node_modules/@react-navigation/native/lib/commonjs/index.js
@@ -127,3 +127,7 @@ Object.keys(_core).forEach(function (key) {
 });
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 //# sourceMappingURL=index.js.map
+Object.defineProperty(exports, 'useLocale', {
+  enumerable: true,
+  get: function () { return function useLocale() { return { direction: 'ltr' } } }
+})
diff --git a/node_modules/@react-navigation/native/lib/module/index.js b/node_modules/@react-navigation/native/lib/module/index.js
index c596c2b..3a0fba7 100644
--- a/node_modules/@react-navigation/native/lib/module/index.js
+++ b/node_modules/@react-navigation/native/lib/module/index.js
@@ -13,3 +13,4 @@ export { default as useLinkTo } from './useLinkTo';
 export { default as useScrollToTop } from './useScrollToTop';
 export * from '@react-navigation/core';
 //# sourceMappingURL=index.js.map
+export function useLocale() { return { direction: 'ltr' } }
